benchmark/streams/writable-manywrites.js

```diff
@@ -4,26 +4,43 @@ const common = require('../common');
 const Writable = require('stream').Writable;
 
 const bench = common.createBenchmark(main, {
-  n: [2e6],
-  sync: ['yes', 'no']
+  n: [1e6],
+  sync: ['yes', 'no'],
+  writev: ['yes', 'no'],
+  callback: ['yes', 'no']
 });
 
-function main({ n, sync }) {
+function nop() {
+
+}
+
+function main({ n, sync, writev, callback }) {
   const b = Buffer.allocUnsafe(1024);
   const s = new Writable();
   sync = sync === 'yes';
-  s._write = function(chunk, encoding, cb) {
-    if (sync)
-      cb();
-    else
-      process.nextTick(cb);
-  };
+
+  if (writev === 'yes') {
+    s._writev = function(chunks, cb) {
+      if (sync)
+        cb();
+      else
+        process.nextTick(cb);
+    };
+  } else {
+    s._write = function(chunk, encoding, cb) {
+      if (sync)
+        cb();
+      else
+        process.nextTick(cb);
+    };
+  }
 
   bench.start();
 
+  let cb = callback === 'yes' ? nop : null;
```

Comment:
Nit: less text and clearer IMO
```suggestion
  let cb = callback === 'yes' ? () => {} : null;
```